### PR TITLE
Schedule cron for CI to run Scripts

### LINKED ISSUE #19: CI Pipeline for rotating 24 hours running
Find a solution to make our software run every 24 hours with a 2 minute time out per file execution. Each ruby file will need to be divided up into its own pipeline.

### PR DESCRIPTION
**Pull request by awasum**
First making it run every 30 minutes so I see success early enough. This will fix #19

### CODE REVIEW

File: .github/workflows/schedule_CI.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+      - run: ruby getHomeBrewJson.rb
+      - run: ruby getRepologyJson.rb
+
```
Comment (BaffourAdu) [inline .github/workflows/schedule_CI.yml:15]: I suggest we add the comparison script to the scripts too

Comment (awasum) [reply .github/workflows/schedule_CI.yml:15]: Good idea, I will test and add it.

Comment (awasum) [reply .github/workflows/schedule_CI.yml:16]: @BaffourAdu I have updated this PR. I think its now adjusted wrt your feedback. Thanks

**Comment by awasum** [comment]
> Looks good. Nice job!

Thanks @BaffourAdu . You also did great on this project.

**Pull request merged**